syft/workers/base.py

```diff
@@ -193,15 +193,15 @@ def send(self, tensor, workers, ptr_id=None):
             ptr_id = int(10e10 * random.random())
 
         # Send the object
-        self.send_obj((ptr_id, tensor), worker)
+        self.send_obj(tensor, worker)
 
         pointer = tensor.create_pointer(
-            owner=self, location=worker, id_at_location=ptr_id, register=True
+            owner=self, location=worker, id_at_location=tensor.id, register=True, ptr_id=ptr_id
         )
 
         return pointer
 
-    def set_obj(self, obj_data):
+    def set_obj(self, obj):
         """This adds an object to the registry of objects.
 
         :Parameters:
```

Comment:
should this section be in the new style or is that beyond the scope of this pr

Reply:
beyond the scope I think

Reply:
i figured style stuff could be multiple PRs (collaborative) - i fixed a few doc style things and a few other things as well.

Reply:
Sounds good